assets/src/edit-story/components/panels/mediaStyle.js

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Background opacity should be initialized to 100% for all new image elements in [the defaultAttributes object](https://github.com/google/web-stories-wp/blob/41a0f3049334ab063a9f6612ff9b28452d29cfb5/assets/src/edit-story/elements/image/index.js#L29-L33).